js_modules/dagster-ui/packages/ui-core/src/assets/AssetEventMetadataPlots.tsx

```diff
@@ -20,12 +20,11 @@ export const AssetTimeMetadataPlots = ({
   asSidebarSection?: boolean;
   columnCount?: number;
 }) => {
-  const {materializations, observations, loading} = useRecentAssetEvents(
-    assetKey,
-    limit,
-    MaterializationHistoryEventTypeSelector.MATERIALIZATION,
-  );
-  const grouped = useGroupedEvents('time', materializations, observations, undefined);
+  const {events, loading} = useRecentAssetEvents(assetKey, limit, [
```

Comment:
we might want to increase the limit here - previously we were fetching `limit` materializations and `limit` observations and then grouping them, now we are just fetching `limit` materializations and observations total. @salazarm @bengotow  what do you think?

Reply:
going to punt on this for now. we can up the limit later if needed